webdriver-spec.html

```diff
@@ -676,19 +676,21 @@
  <dt><dfn data-lt="intermediary nodes">Intermediary node</dfn>
  <dd>Intermediary nodes are those that act as proxies,
   implementing both the client and server sides of the <a href=#protocol>protocol</a>.
-  Intermediary nodes must be black-box indistinguishable from a <a>remote end</a>
-  from the point of view of <a>local end</a>,
-  and so are bound by the requirements on a <a>remote end</a> in terms of the wire protocol.
   However they are not expected to implement <a>remote end steps</a> directly.
 
  <dt><dfn>Endpoint node</dfn>
  <dd>An endpoint node is the final <a>remote end</a>
   in a chain of nodes that is not an <a>intermediary node</a>.
   The endpoint node is implemented by a user agent or a similar program.
-  An endpoint node must be, like <a>intermediary nodes</a>,
-  indistinguishable from a <a>remote end</a>.
 </dl>
 
+<p> Both <a data-lt="intermediary nodes">intermediary</a> and
+  <a data-lt="endpoint node">endpoint</a> nodes must be black-box
+  indistinguishable from a <a>remote end</a>,
```

Comment:
Because both `intermediary` and `endpoint` nodes are types of `node types`, it's probably enough to say:

```all remote end node types must be black-box indistinguishable from the point of view of local ends, and so...```

This formulation allows to add additional node types if we ever go crazy and do that.